### DISCUSSION TITLE
personal site on InterServer - cannot "save" any settings

### DISCUSSION THREAD

**Discussion by wcrisafu** [oqtane/oqtane.framework#2437] [Q&A] [answered]
Hi,

I finally decided to deploy Oqtane 3.2 as my personal website. I downloaded the latest version (downloaded yesterday's with the latest updates + recompiled) and used WebDeploy. Everything worked great, I was able to start the site, configure the SQL DB. The problem started immediately after. I can login and I can create a simple page but now i cannot change any of the settings:
1) i can add the html module but I cannot change the html module title
2) I cannot change the site icon
3) I cannot set a new homepage

Any idea what I have done wrong or where I could look for more details?

Here is an example error while trying to change the page permission.
Event Error:
Template: Error Saving Page {Page} {Error}
Message: Error Saving Page null Object reference not set to an instance of an object.
Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at Oqtane.Modules.Admin.Pages.Edit.SavePage() in D:\SourceCode\Oqtane32\oqtane.framework\Oqtane.Client\Modules\Admin\Pages\Edit.razor:line 515
Properties: {"Page":null,"Error":"Object reference not set to an instance of an object."}

**Comment by leigh-pointer** [comment, marked as answer]
@wcrisafu It sounds like you have webDAV enabled.  The best option for this is to update your web.config.  Check out https://github.com/oqtane/oqtane.framework/discussions/1207

**Reply by wcrisafu** [reply]
You were 100% correct!. I applied your setting and it started working immediately. I will try to read more about this issue with WebDAV. 

Thank you again!

**Reply by leigh-pointer** [reply]
pleased it helped.
